packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/TaskNavButtons/TaskNavButtonsContainer.spec.js

```diff
@@ -214,24 +188,29 @@ describe('TaskNavButtonsContainer', function () {
 
     afterEach(function () {
       completeClassificationSpy.resetHistory()
-      createDefaultAnnotationIfThereIsNoneSpy.resetHistory()
+      addAnnotationSpy.resetHistory()
       onSubmitSpy.resetHistory()
       preventDefaultSpy.resetHistory()
     })
 
     after(function () {
-      createDefaultAnnotationIfThereIsNoneSpy.restore()
+      addAnnotationSpy.restore()
       onSubmitSpy.restore()
     })
 
-    it('should prevent the event default', function () {
+    it('should create a default annotation for each task if there is not an annotation for that task', function () {
       wrapper.instance().onSubmit({ preventDefault: preventDefaultSpy })
-      expect(preventDefaultSpy).to.have.been.calledOnce()
+      const classification = classificationStore.active.toJSON()
+
+      tasks.forEach((task) => {
+        expect(addAnnotationSpy.withArgs(null, task)).to.have.been.calledOnce()
```

Comment:
Outside the scope of this PR, but a better test here would be to expect that the annotation for this task, stored on the classification, is the default annotation.

Reply:
Nevermind, that's the next check. This expectation can be deleted. We don't care how the default annotation is implemented, only that it's created. We want to test behaviour, not implementation.